docs/documentation/features/internationalization.md

```diff
@@ -132,6 +132,182 @@ channel-type.yahooweather.temperature.option.OPTION2 = Option Nummer 2
+thing-type.config.weatherunderground.weather.refresh.label = Fréquence de rafraîchissement
+thing-type.config.weatherunderground.weather.refresh.description = La fréquence de rafraîchissement des données en minutes.
+
+# channel group types
+channel-group-type.weatherunderground.current.label = Météo actuelle
+channel-group-type.weatherunderground.current.description = La météo actuelle.
+channel-group-type.weatherunderground.forecast.label = Météo prévue
+channel-group-type.weatherunderground.forecast.description = La météo prévue.
+
+# channel groups
+thing-type.weatherunderground.weather.group.forecastTomorrow.label = Météo de demain
+thing-type.weatherunderground.weather.group.forecastTomorrow.description = La météo prévue demain.
+thing-type.weatherunderground.weather.group.forecastDay2.label = Météo dans 2 jours
+thing-type.weatherunderground.weather.group.forecastDay2.description = La météo prévue dans 2 jours.
+
+# channel types
+channel-type.weatherunderground.currentConditions.label = Conditions actuelles
+channel-type.weatherunderground.currentConditions.description = Les conditions météo actuelles.
+channel-type.weatherunderground.temperature.label = Température
+channel-type.weatherunderground.temperature.description = La température actuelle.
+channel-type.weatherunderground.maxTemperature.label = Température maximale
+channel-type.weatherunderground.maxTemperature.description = La température maximale.
+
+# channel type configuration
+channel-type.config.weatherunderground.temperature.SourceUnit.label = Unité de température
+channel-type.config.weatherunderground.temperature.SourceUnit.description = Choix de l'unité de température fournie par le service Weather Underground pour la température actuelle.
+channel-type.config.weatherunderground.temperature.SourceUnit.option.C = Degrés Celsius
+channel-type.config.weatherunderground.temperature.SourceUnit.option.F = Degrés Fahrenheit
+channel-type.config.weatherunderground.maxTemperature.SourceUnit.label = Unité de température maximale
+channel-type.config.weatherunderground.maxTemperature.SourceUnit.description = Choix de l'unité de température fournie par le service Weather Undergroundde pour la température maximale.
+channel-type.config.weatherunderground.maxTemperature.SourceUnit.option.C = Degrés Celsius
+channel-type.config.weatherunderground.maxTemperature.SourceUnit.option.F = Degrés Fahrenheit
+
+```
+
+So the key for referencing a label of a defined thing type is `thing-type.<binding-id>.<thing-type-id>.label`. The label of a channel type can be referenced with `channel-type.<binding-id>.<channel-type-id>.label`, the label of a channel group type with `channel-group-type.<binding-id>.<channel-group-type-id>.label` and the label of a channel group with `thing-type.<binding-id>.<thing-type-id>.group.<channel-group-id>.label`. The config description parameter key is `thing-type.config.<binding-id>.<thing-type-id>.<parameter-name>.label`.
```

Comment:
I think the example from Yahoo Weather could be simply removed because the example with Weather Underground covers I think all translation cases you can find in the Yahoo Weather.
WDYT ?